backend/coins/btc/account.go

```diff
@@ -434,6 +436,9 @@ outer:
 
 // Balance implements the interface.
 func (account *Account) Balance() *accounts.Balance {
+	if account.fatalError {
+		return accounts.NewBalance(coin.NewAmountFromInt64(0), coin.NewAmountFromInt64(0))
```

Comment:
I'm not a fan of "lying" to callers. I would prefer "proper" design by contract.

Reply:
done